### DISCUSSION TITLE
Extend xrpl.org's toml lookup to do account-based lookup

### DISCUSSION THREAD

**Discussion by mDuo13** [XRPLBounties/Proposals#62] [Closed (claimed or cancelled bounty)]
The [xrp-ledger.toml checker on XRPL.org](https://xrpl.org/xrp-ledger-toml-checker.html) is a handy tool for looking up what accounts are claimed by a specific domain (including domain verification), but it could be handy to be able to do it the other way around and check to see if an account really is claimed by the domain it has set in its `Domain` field.

So in addition to being able to put in a domain and verify the toml from there, you could put in an account to verify that account's domain: if it has a `Domain` field, you look up to see if that domain serves a toml file and if that toml claims ownership of the account.

Upstream ticket: https://github.com/XRPLF/xrpl-dev-portal/issues/576

**Comment by jonathanlei** [comment]
This bounty has been completed.

~This idea has been proposed as a Bounty and approved! See details here: https://github.com/XRPLBounties/Proposals/blob/main/open_bounties/0062%20Account-based%20toml%20Lookup.md~

**Comment by ripci504** [comment]
I have submitted a pull request for this bounty. Would love feedback on it

https://github.com/XRPLF/xrpl-dev-portal/pull/1644

**Reply by jonathanlei** [reply]
Thanks! Left comment on PR on step 2 (testing)

**Reply by ripci504** [reply]
How do you recommend I write tests? Should I make another file with the script so it can be run by NODE JS outside of the browser?

**Reply by jonathanlei** [reply]
Yes that would be fine!
